WalletWasabi.Fluent/Behaviors/KeyDownTrigger.cs

```diff
@@ -0,0 +1,51 @@
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Avalonia;
+using Avalonia.Input;
+using Avalonia.Interactivity;
+using Avalonia.Xaml.Interactivity;
+using ReactiveUI;
+using WalletWasabi.Fluent.Extensions;
+
+namespace WalletWasabi.Fluent.Behaviors;
+
+public class KeyDownTrigger : Trigger
+{
+	public static readonly StyledProperty<RoutingStrategies> EventRoutingStrategyProperty = AvaloniaProperty.Register<KeyDownTrigger, RoutingStrategies>(nameof(EventRoutingStrategy));
+
+	public static readonly StyledProperty<Key> KeyProperty = AvaloniaProperty.Register<KeyDownTrigger, Key>(nameof(Key));
+	private readonly CompositeDisposable _disposables = new();
+
+	public RoutingStrategies EventRoutingStrategy
+	{
+		get => GetValue(EventRoutingStrategyProperty);
+		set => SetValue(EventRoutingStrategyProperty, value);
+	}
+
+	public Key Key
+	{
+		get => GetValue(KeyProperty);
+		set => SetValue(KeyProperty, value);
+	}
+
+	protected override void OnAttached()
+	{
+		base.OnAttached();
+
+		if (AssociatedObject is InputElement interactive)
+		{
+			this.WhenAnyValue(x => x.EventRoutingStrategy, x => x.Key)
```

Comment:
https://github.com/zkSNACKs/WalletWasabi/blob/acbb6f5ad1591b55ed70864ecdc5c6e3160c9c83/WalletWasabi.Fluent/Behaviors/ExecuteCommandOnKeyDownBehavior.cs#L68

Reply:
What do you mean?